### PR TITLE
Adding iclamp example

### PR DESCRIPTION
**Pull request by kaeldai**

### CODE REVIEW

File: examples/5_cells_iclamp/build_network.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import numpy as np
+from neuron import h
+from bmtk.builder import NetworkBuilder
+
+cell_models = [
+    {
+        'model_name': 'Scnn1a', 'positions': [(0.0, 0.0, 0.0)], 'ei': 'e', 'morphology_file': 'Scnn1a_473845048_m.swc',
```
Comment (hernando) [inline examples/5_cells_iclamp/build_network.py]: Use morphology instead of morphology_file

File: examples/5_cells_iclamp/output/config.json [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+    }
+  }, 
+  "run": {
+    "nsteps_block": 10000, 
+    "dt": 0.1, 
+    "dL": 20.0, 
+    "spike_threshold": -15, 
+    "tstop": 4000.0
+  }, 
+  "node_sets": {
```
Comment (hernando) [inline examples/5_cells_iclamp/output/config.json]: Same comment as before

File: examples/5_cells_iclamp/output/spikes.csv [hunk 3]
```diff
@@ -0,0 +1,126 @@
+533.0 2
```
Comment (hernando) [inline examples/5_cells_iclamp/output/spikes.csv]: This CSV is not part of the spec. Spikes must be stored as in H5.

File: examples/5_cells_iclamp/build_network.py (hunk 1 above)
Comment (hernando) [reply examples/5_cells_iclamp/build_network.py]: And remove .swc extension

**Comment by kaeldai** [comment]
I am closing this as I created another PR, #36 that does the same thing and address all the issues above

**Pull request closed without merging**